### PR TITLE
Update readme.md

### PR DESCRIPTION
**Pull request by danyaljj**

### CODE REVIEW

**Comment by danyaljj** [comment]
FYI @nalourie-ai2

File: readme.md [hunk 1]
```diff
@@ -213,7 +213,7 @@ To create a batch, write a batch definition (see the
 [example batch definition][example-batch-definition]), create some data
 in the [JSON Lines][json-lines] format, and then create the batch using
 `amti create-batch`. Use the `-h` option for details. You can find some
-example data in the [`data.jsonl`][data-jsonl] file.
```
Comment (nalourie-ai2) [inline readme.md]: This link is actually reference-style (see [here](https://daringfireball.net/projects/markdown/syntax#link) in the markdown spec). So, the place to fix the broken link would be line 227: `[data-jsonl]: ./examples/test/data.jsonl`.

**Comment by nalourie-ai2** [comment]
Thanks so much for the contribution @danyaljj! Currently, the change would still leave the link broken (as explained in my comment above). If you push a new commit that instead fixes the link at line 227, then I can go ahead and merge it.

**Comment by nalourie-ai2** [comment]
@danyaljj, since I don't have write access to your fork, I'm going to close this PR and open a new one to fully fix the link and get these changes merged.

Thanks again for catching this bug!

**Pull request closed without merging**